text/symbols.py

```diff
@@ -26,12 +26,12 @@
 _moh_punctuation = "!(),.;? "
 _special = "-"
 _letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
-_silences = ["@sp", "@spn", "@sil"] # ["sp", "spn", "sil", "@sp", "@spn", "@sil"] can be used as well
+_silences = ["sp", "spn", "sil"] # ["sp", "spn", "sil", "@sp", "@spn", "@sil"] can be used as well
```

Comment:
This matches (at least) the short pause tokens in ming024's MFA TextGrids. Using `@sp` etc. lead to text/pitch sequence length mismatches when training a new LJ model.